extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -196,12 +202,81 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 				}
 			}
 		});
-
 		this._assessmentResults = {
 			issues: serverLevelAssessments,
 			databaseAssessments: databaseLevelAssessments
 		};
 
+		try {
+			const queryProvider = azdata.dataprotocol.getProvider<azdata.QueryProvider>(connectionProfile.providerId, azdata.DataProviderType.QueryProvider);
+			const query = 'SELECT name, compatibility_level FROM sys.databases; ';
+			const results = await queryProvider.runQueryAndReturn(await (azdata.connection.getUriForConnection(this.sourceConnectionId)), query);
+			const compatLevelHashMap: Map<string, string> = new Map();
+			results.rows.forEach(r => {
+				compatLevelHashMap.set(r[0].displayValue, r[1].displayValue);
+			});
+
+			sendSqlMigrationActionEvent(
+				TelemetryViews.MigrationWizardTargetSelectionPage,
+				TelemetryAction.ServerAssessment,
+				{
+					'sessionId': this._sessionId,
+					'tenantId': this._azureAccount.properties.tenants[0].id,
+					'accountId': this._azureAccount.key.accountId,
+					'hashedServerName': hashString((connectionProfile.serverName))
+				},
+				{
+					'issuesCount': this._assessmentResults.issues.length,
+					'duration': endTime - startTime,
+					'databaseCount': this._assessmentResults.databaseAssessments.length,
+					'startTime': startTime,
+					'endTime': endTime,
+				}
+			);
+
+			this._assessmentResults.issues.forEach(i => {
```

Comment:
do assessment exceptions show up in the issues list where the assessment tool was unable to assess a database / instance

Reply:
No, it does not. I will have to expose that from the backend